Dockerfile

```diff
@@ -70,9 +77,8 @@ RUN Rscript -e "\
       paste0('$import_url', '$import_tag'), \
       paste0('$utils_url', '$utils_tag') \
     ); \
-  workflow_pkgs <- renv::dependencies('DESCRIPTION')[['Package']]; \
-  workflow_pkgs <- grep('^pacta[.]', workflow_pkgs, value = TRUE, invert = TRUE); \
-  pak::pak(c(gh_pkgs, workflow_pkgs)); \
+  print(gh_pkgs); \
+  pak::pak(c(gh_pkgs)); \
```

Comment:
I'm skeptical of trying to overwrite the previously installed PACTA packages with different explicitly tagged versions, but if you're confident with it, that's fine... I would just worry a bit that it might not go exactly as expected.